packages/builder/src/components/grants/About.tsx

```diff
@@ -34,8 +39,46 @@ export default function About({
       chainId,
       projectID: params.id!,
       applications,
+      rounds: state.rounds,
     };
   });
+  const [applicationsToShow, setApplicationsToShow] = useState<Application[]>(
+    []
+  );
+
+  useEffect(() => {
+    const roundToChain = new Map<string, number>();
+    props.applications.forEach((application) => {
+      roundToChain.set(application.roundID, application.chainId);
+    });
+    dispatch(loadRounds(roundToChain));
+  }, [dispatch, props.applications]);
+
+  useEffect(() => {
```

Comment:
we could do `const appsToShow = useMemo(..)` here instead of `useEffect + useState`